randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -258,34 +258,41 @@ Extra - room_id: [9]
               # Stand on Frozen Reo
               Combat (Beginner) and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
   > Door to Maintenance Annex
+      After Sector 2 Zoro Zig-Zag Bomb Blocks Broken and Can Bounce in Ball
+  > Event - Break Bomb Blocks
       Can Bounce in Ball and Can Use Any Bombs
 
 > Door to Maintenance Annex; Heals? False
   * Layers: default
   * L0 Hatch to Maintenance Annex/Door to Zoro Zig-Zag
   * Extra - door_idx: (21,)
   > Door to Data Hub
+      All of the following:
+          Morph Ball and After Sector 2 Zoro Zig-Zag Bomb Blocks Broken
+          Any of the following:
+              Space Jump
+              Hi-Jump and Wall Jump (Intermediate)
+  > Event - Break Bomb Blocks
       All of the following:
           Morph Ball
           Any of the following:
-              All of the following:
-                  Space Jump
-                  Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Expert)
-                              Screw Attack and Knowledge (Beginner)
-              All of the following:
-                  Hi-Jump
-                  Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Advanced)
-                              Screw Attack and Knowledge (Beginner) and Wall Jump (Intermediate)
+              # Climbing Requirement
+              Space Jump
+              Hi-Jump and Wall Jump (Intermediate)
+          Any of the following:
+              # Break Blocks
+              Can Use Power Bombs
+              Screw Attack and Knowledge (Beginner)
+              # Cursed Method: https://youtu.be/Cqf5mZg1NZc
+              Hi-Jump and Jump Bombjump (Hypermode) and Mid-Air Morph (Intermediate) and Can Use Bombs
```

Comment:
You broke the bomb blocks in the video with spring ball. I assume that laying a bomb -> get hit by explosion -> lay bomb at the peak also works to destroy that block?

Reply:
not quite sure what you mean by this. The first bomb block was destroyed exactly as you explained, it wasn't done with just spring ball.
If you are implying that these should be separated or something, I don't think thats really worth doing, considering both methods are still technically Jump Bomb Jumps

Reply:
sorry, meant to clarify that one specific bomb block, but forgot xP

Reply:
oh I see now what you mean. I guess maybe you could? I tried it for a little bit and couldn't get it though.

It shouldn't really change anything considering the fact that you need hi jump anyway for the last JBJ.

Reply:
If you explicitly need spring ball, please mark it by using the template.

Reply:
added